sdk/storage/azure-storage-common/CHANGELOG.md

```diff
@@ -10,6 +10,8 @@
 
 ### Other Changes
 
+- Added support for ICU 75.1 or later.
```

Comment:
"We [...] disable ICU C++ API [...] because we don't need ICU C++ API".

Do we then "Add[ed] support for ICU 75.1 [...]" or "Fix[ed] compilation with newest versions of `libxml2` due to ICU API 75.1+ requiring C++17."?

Reply:
If we mark this as a bug fix, the "Fix[ed] compilation ..." style will be better.

I'm not familiar with how Azure SDK for C++ decides which is a bug fix... Do you think that this is a bug fix?

FYI: libxml2 version isn't related in this context. Not newest versions of libxml2 can use ICU 75.1 or later. So the version of ICU that is used by libxml2 is only important.

Reply:
Thank you. I honestly don't know about the context to the full extent.
You are the customer, you use ICU, you are more familiar with it, and you see it as a feature - I lean towards phrasing it your way then.

Reply:
Thanks for confirming this.

I've applied all your suggestions. Thanks!